rds-update-rdsh-collection/azuredeploy.json

```diff
@@ -0,0 +1,318 @@
+                "Standard_DS1",
+                "Standard_DS2",
+                "Standard_DS3",
+                "Standard_DS4",
+                "Standard_DS11",
+                "Standard_DS12",
+                "Standard_DS13",
+                "Standard_DS14",
+                "Standard_G1",
+                "Standard_G2",
+                "Standard_G3",
+                "Standard_G4",
+                "Standard_G5",
+                "Standard_GS1",
+                "Standard_GS2",
+                "Standard_GS3",
+                "Standard_GS4",
+                "Standard_GS5"
+            ]
+        },
+
+        "userLogoffTimeoutInMinutes": {
+            "type": "int",
+            "defaultValue": 60,
+            "metadata": {
+                "description": "Timeout before forced user logoff from old RDSH instances"
+            }
+        },
+
+        "existingDomainName": {
+            "type": "string",
+            "metadata": {
+                "description": "Domain name of the exisiting RDS deployment"
+            }
+        },
+        "existingAdminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of administrator in the existing RDS deployment"
+            }
+        },
+        "existingAdminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Domain administrator password"
+            }
+        },
+
+        "existingVnetName": {
+            "type": "string",
+            "defaultValue": "vnet",
+            "metadata": {
+                "description": "VNET name"
+            }
+        },
+        "existingSubnetName": {
+            "type": "string",
+            "defaultValue": "Subnet",
+            "metadata": {
+                "description": "Subnet name"
+            }
+        },
+
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+            },
+            "defaultValue": "https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/rds-update-rdsh-collection/"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+            },
+            "defaultValue": ""
+        }
+
+    },
+
+    "variables": {
+
+        "iteration": "[toLower(parameters('rdshUpdateIteration'))]",
+
+        "assetLocation": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/rds-deployment/",
```

Comment:
remove this and use _artifactsLocation - all references to artifacts should use the same pattern.